components/Careers/AllJobsPage.vue

```diff
@@ -0,0 +1,427 @@
+                                        </div>
+    
+                                        <div class="filter-search-form relative filter-border flex justify-center items-center">
+                                            <i class="uil uil-map-marker icons"></i>
+                                            <select v-model="currentdata.location" class="form-select" data-trigger name="choices-location" id="choices-location" aria-label="Default select example">
+                                                <option value="none" selected>Select</option>
+                                                <option v-for="location in sortedLocations" :value="location">{{ location }}</option>
+                                            </select>
+                                        </div>
+    
+                                        <div class="filter-search-form relative filter-border flex justify-center items-center">
+                                            <i class="uil uil-briefcase-alt icons"></i>
+                                            <select v-model="currentdata.type" class="form-select " data-trigger name="choices-type" id="choices-type" aria-label="Default select example">
+                                                <option value="none" selected>Select</option>
+                                                <option v-for="location in sortedTypes" :value="location">{{ location }}</option>
+                                            </select>
+                                        </div>
+    
+                                        <input type="submit" @click.prevent="subitHandler()" id="search" name="search" style="height: 60px;" class="btn bg-indigo-600 hover:bg-indigo-700 border-indigo-600 hover:border-indigo-700 text-white searchbtn submit-btn w-100" value="Search">
+                                    </div>
+                                    <!--end grid-->
+                                </div>
+                                <!--end container-->
+                            </form>
+                        </div>
+                    </div>
+                    <!--ed col-->
+                </div>
+                <!--grid-->
+            </div>
+            <!--end container-->
+            <!-- form container ends -->
+
+    
+            <div class="container">
+                <div class="grid lg:grid-cols-3 md:grid-cols-2 grid-cols-1 mt-8 gap-[30px]">
+    
+                    <SingleJobCard v-for="job in getPaginatedData()" :data="job" />
+    
+                </div>
+                <!--end grid-->
+    
+            </div>
+            <!--end container-->
+    
+            <div class="grid md:grid-cols-12 grid-cols-1 mt-8">
+                <div class="md:col-span-12 text-center">
+                    <nav aria-label="Page navigation example">
```

Comment:
if there is no job for particular search, then remove the pagination section

Reply:
done